lms/djangoapps/lti_provider/tests/test_outcomes.py

```diff
@@ -363,3 +299,128 @@ def test_response_with_failing_status_field(self):
             major_code='<imsx_codeMajor>failure</imsx_codeMajor>'
         )
         self.assertFalse(outcomes.check_replace_result_response(response))
+
+
+class TestAssignmentsForProblem(ModuleStoreTestCase):
```

Comment:
If you're only setting up one course tree to be reused across your tests, please use [`SharedModuleStoreTestCase`](http://swampcastle.org/2015/07/28/writing-faster-modulestore-tests.html) instead.